yarn.lock

```diff
@@ -13789,6 +13781,12 @@ has-ansi@^2.0.0:
   dependencies:
     ansi-regex "^2.0.0"
 
+has-binary2@cypress-io/has-binary#8580a33df21e8b36a43f57872a82c60829636a92:
+  version "1.0.2"
+  resolved "https://codeload.github.com/cypress-io/has-binary/tar.gz/8580a33df21e8b36a43f57872a82c60829636a92"
+  dependencies:
+    isarray "2.0.1"
+
```

Comment:
@bahmutov Why is this included in your lockfile? Get nervous seeing `has-binary2` pop up again.

Reply:
:scream: 

@bahmutov I believe the final fix for this is to `rm -rf ~/.cache/yarn`

I am still rattled that `yarn` can non-deterministically generate a lockfile based on whatever random dependencies you have cached locally... that seems like a problem...